content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/cordapp-packaging.md

```diff
@@ -1,30 +1,30 @@
 ---
-date: '2021-09-21'
+date: '2021-11-14'
 title: "CorDapp Packaging"
-draft: true
 menu:
   corda-5-alpha:
-    parent: corda-5-alpha-tutorials-dev
-    identifier: corda-5-alpha-tutorial-dev-one
-    weight: 1000
+    parent: corda-5-alpha-tutorials-deploy
+    identifier: corda-5-alpha-tutorial-deploy-cordapp-packaging
+    weight: 2000
 section_menu: corda-5-alpha
 ---
 
-This section describes how to package your CorDapp as CPKs, CPBs, and CPIs. You can read more about the CorDapp packaging format in the [Key concepts](../../introduction/key-concepts.html#packaging) section.
+This section describes how to package your CorDapp as CPKs, CPBs, and CPIs.
 
 ## Before You Start
 
-You must install [Corda CLI](../corda-cli/overview.html).
+You will need a working copy of `corda-cli` with the MGM and package plugins installed. 
```

Comment:
```suggestion
You will need a working copy of the Corda CLI with the MGM and package plugins installed. 
```
Also, may not need to specify "with the MGM and package plugins installed" as I think they are now installed by default..

Reply:
Will I take "with the MGM and package plugins installed" out?